bindings_ffi/src/mls.rs

```diff
@@ -5391,6 +5391,40 @@ mod tests {
         assert!(stream.is_closed());
     }
 
+
+    #[tokio::test(flavor = "multi_thread", worker_threads = 5)]
+    async fn test_stream_all_messages_error() {
```

Comment:
if you write the tests in the style of tests in https://github.com/xmtp/libxmtp/blob/main/xmtp_mls/src/subscriptions/stream_all.rs, does it still reproduce?